_delphi_utils_python/delphi_utils/utils.py

```diff
@@ -95,5 +95,32 @@ def transfer_files():
     delivery_dir = params["delivery"].get("delivery_dir", None)
     files_to_export = os.listdir(export_dir)
     for file_name in files_to_export:
-        if file_name.endswith(".csv") or file_name.endswith(".CSV"):
-            move(os.path.join(export_dir, file_name), delivery_dir)
+        if file_name.lower().endswith(".csv"):
+            move(os.path.join(export_dir, file_name), os.path.join(delivery_dir, file_name))
+
+def delete_move_files():
+    """Delete or move output files depending on dir settings provided in params.
+
+    1. Delete files in export-dir if delivery-dir is specified and is different
+       from export_dir (aka only delete files produced by the most recent run)
+    2. If validation-failures-dir is specified, move failed files there instead
+    """
+    params = read_params()
+    export_dir = params["common"].get("export_dir", None)
+    delivery_dir = params["delivery"].get("delivery_dir", None)
```

Comment:
Add early exit here for readability and to reduce empty-directory litter. something like
```suggestion
    delivery_dir = params["delivery"].get("delivery_dir", None)
    if delivery_dir is None:
        return
```

Reply:
Yep, seems correct. 
Should signals be allowed to be not on dry run, but not have a delivery dir?
If yes, then deleting files would be bad.
I don't think 4 should ever exist technically, so we might want to throw an assertion exception there too?